libpetrifui/dish_file.c

```diff
@@ -21,7 +21,7 @@
 #include "dish_file.h"
 
 #include <libxml/parser.h>
-#include <openssl/sha.h>
+#include <gcrypt.h>
```

Comment:
Is this line necessary? No function in `openssl/sha.h` is used in the code, and `gcrypt.h` should not be necessary, as `file_ops_dir_to_hash ()` is not called directly in this file.